src/consensus/vote.cpp

```diff
@@ -453,12 +467,11 @@ void VoteManager::cleanupVotes(uint64_t pbft_round) {
   for (auto const& v_hash : remove_verified_votes_hash) {
     db_->removeVerifiedVoteToBatch(v_hash, batch);
   }
-
   db_->commitWriteBatch(batch);
 }
 
-bool VoteManager::voteValidation(taraxa::Vote& vote, size_t const dpos_total_votes_count,
-                                 size_t const sortition_threshold) const {
+bool VoteManager::voteValidation(taraxa::Vote& vote, size_t const& dpos_total_votes_count,
```

Comment:
same here, there is not need for const & for size_t args

Reply:
Fixed